src/scss/grommet-core/_objects.heading.scss

```diff
@@ -0,0 +1,17 @@
+.heading {
+  margin-bottom: halve($inuit-base-spacing-unit);
+
+  &--large {
+    font-size: 60px;
```

Comment:
I would think the large and small would be relative to the tag chosen. Is that not the case, that setting large or small fixes the font size irrespective of the tag?